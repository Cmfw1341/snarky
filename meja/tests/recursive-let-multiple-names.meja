let rec (x, y) = (12, true);
