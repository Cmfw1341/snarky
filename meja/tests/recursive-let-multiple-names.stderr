File "tests/recursive-let-multiple-names.meja", line 1, characters 9-13:
Error: This pattern is not allowed for recursive bindings:
too many variables are bound.
